app/(main)/page.tsx

```diff
@@ -281,9 +290,16 @@ export default function HomePage() {
             onChange={handleCategory}
             className="overflow-x-auto md:w-auto lg:w-fit"
           >
-            <ToggleGroupOption size="md" value={''} className="font-semibold">
+            <ToggleGroupOption size="md" value="" className="font-semibold">
               All
             </ToggleGroupOption>
+            <ToggleGroupOption
+              size="md"
+              value="devconflict"
+              className="bg-[url('/devconflict/dd-tiger.svg')] bg-cover bg-center font-semibold text-outline-primary-high-em"
```

Comment:
bg isn't being applied, right? If it is, I can't notice.

Reply:
it should have been removed.